lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -106,11 +115,18 @@ def redcap_det_scan(*, db: DatabaseSession, cache: TTLCache, det: dict, redcap_r
     # By verifying illness_questionnaire is complete first, we minimize the
     # delay in data ingestion since the back_end_mail_scans is completed the day after enrollment.
     #   -Jover, 29 June 2020
-    if not redcap_record['illness_q_date'] and not is_complete('back_end_mail_scans', redcap_record):
+
+    # Add check for `illness_questionnaire` is complete because the new
+    # SCAN In-Person Enrollment project does not have the `illness_q_date` field
+    # and it does not have the `back_end_mail_scans` instrument.
+    #   -Jover, 16 July 2020
+    if (not redcap_record.get('illness_q_date') and
```

Comment:
Would projects other than the new SCAN In-Person Enrollment project still be handled the same way with this change? E.g. for other projects, would records with incomplete enrollment which would have been previously caught by `not redcap_record['illness_q_date'] and not is_complete('back_end_mail_scans', redcap_record)` always have incomplete illness_questionnaire and still be caught by the new logic?

Reply:
By suggestion from @tsibley , I've changed the structure of the If statement logic to now its `not(...or...or...)`. I think that clarifies the old projects would still get caught.